brion/morphology.cpp

```diff
@@ -69,27 +69,27 @@ CellFamily Morphology::getCellFamily() const
     return _impl->plugin->getCellFamily();
 }
 
-Vector4fsPtr Morphology::readPoints() const
+Vector4fsPtr Morphology::readPoints()
```

Comment:
Both SWC and ZeroEQ then need to create a copy on each call, otherwise it's equivalent to e.g. ```int& getFoo() const``` which is not correct. For example, the brain morphology transforms the points which has a side effect when applied to a SWC brion morphology.

I decided for this since it's more efficient, explicit and sufficient for the current use cases.